lib/src/main/java/com/diffplug/spotless/pom/SortPomCfg.java

```diff
@@ -27,7 +27,7 @@ public class SortPomCfg implements Serializable {
 
 	public String lineSeparator = System.getProperty("line.separator");
 
-	public boolean expandEmptyElements = true;
```

Comment:
I'm not a maven user - does this affect end-user behavior enough that we should note it in the `CHANGELOG.md`?

Reply:
yes, should be mentioned in the changelog.

Reply:
done.